cmd/retester/retester_test.go

```diff
@@ -32,7 +31,247 @@ func (f *MyFakeClient) GetRef(owner, repo, ref string) (string, error) {
 	return "abcde", nil
 }
 
+func TestLoadConfig(t *testing.T) {
+	c := &Config{
+		Retester: Retester{
+			RetesterPolicy: RetesterPolicy{
+				MaxRetestsForSha: 1, MaxRetestsForShaAndBase: 1,
+			},
+			Oranizations: map[string]Oranization{"openshift": {
+				RetesterPolicy: RetesterPolicy{
+					MaxRetestsForSha: 2, MaxRetestsForShaAndBase: 2, Enabled: true,
+				},
+				Repos: map[string]Repo{
+					"ci-docs": {RetesterPolicy: RetesterPolicy{Enabled: true}},
+					"ci-tools": {RetesterPolicy: RetesterPolicy{
+						MaxRetestsForSha: 3, MaxRetestsForShaAndBase: 3, Enabled: true,
+					}},
+				}},
+			},
+		}}
+	testCases := []struct {
+		name          string
+		file          string
+		expected      *Config
+		expectedError error
+	}{
+		{
+			name:     "basic case",
+			file:     "testdata/testconfig/config.yaml",
+			expected: c,
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			actual, err := loadConfig(tc.file)
+			if diff := cmp.Diff(tc.expectedError, err, testhelper.EquateErrorMessage); diff != "" {
+				t.Errorf("Error differs from expected:\n%s", diff)
+			}
+			if tc.expectedError == nil {
+				if diff := cmp.Diff(tc.expected, actual); diff != "" {
+					t.Errorf("%s differs from expected:\n%s", tc.name, diff)
+				}
+			}
+		})
+	}
+}
+
+func TestGetRetesterPolicy(t *testing.T) {
+	c := &Config{
+		Retester: Retester{
+			RetesterPolicy: RetesterPolicy{MaxRetestsForShaAndBase: 1, MaxRetestsForSha: 1},
+			Oranizations: map[string]Oranization{
+				"openshift": {
+					RetesterPolicy: RetesterPolicy{
+						MaxRetestsForSha: 2, MaxRetestsForShaAndBase: 2, Enabled: true,
```

Comment:
```suggestion
						Enabled: true,
```

Reply:
I wouldn't be able to test all scenarios with this change. I can add more tests.